pandas/core/arrays/base.py

```diff
@@ -219,7 +228,7 @@ def _formatting_values(self):
         # type: () -> np.ndarray
         # At the moment, this has to be an array since we use result.dtype
         """An array of values to be printed in, e.g. the Series repr"""
-        raise AbstractMethodError(self)
+        return np.array(self)
```

Comment:
NO, this should be implemented only by subclasses.

Reply:
Why?

Reply:
This is a very sensible default IMO, subclasses can always override if needed